src/res/translation/translation_de_DE.ts

```diff
@@ -154,7 +154,8 @@
     <message>
         <location filename="../../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Could also stay in english language, since this is also common in german</translatorcomment>
```

Comment:
What is the philosophy for this used in the other translation files?  Is there a common approach?

Reply:
I think this is just a comment and will not be shown in the UI?

Reply:
I guess the question with this item is whether "The Jamulus Development Team" is effectively a name that doesn't need translating, or a description that does.

Reply:
I translated it into german, but IMHO this is something , which one could leave completely in english since it is a kind of common standard ? Isn´t it ?

Reply:
I'd also say: Leave it untranslated.

Reply:
> I think this is just a comment and will not be shown in the UI?

The source will be shown in the UI (if there is no translation), but the translatorcomment will not, of course.

Reply:
I don't have any strong opinion on this. So I don't care if it's translated or not.